rust/lance-file/src/v2/writer.rs

```diff
@@ -42,6 +48,19 @@ pub struct FileWriterOptions {
     pub data_cache_bytes: Option<u64>,
 }
 
+/// Writes Lance files from data
+///
+/// Unlike the reader, the FileWriter is aware of the concept of field ids.  While writing
+/// a batch it will keep track of a field id to column index map which will be returned when
+/// the writer is finished.  This can be used for later reading to map field ids to the correct
+/// column.
+///
+/// The FileWriter can also handle blobs.  The schema should have a field with the metadata
+/// `lance:blob` set to true.  This field should be a struct with two fields: `size` and `position`.
```

Comment:
Is this correct? Doesn't it have a `path` field too?

Reply:
This comment disappeared in the refactor but you are correct.  Probably we will want to add blobs to the table format docs and go over how they work in detail.  I've added a task to the umbrella issue.